### PR TITLE
Fix indentation related issue

### PR DESCRIPTION
**Pull request by rusamentiaga**

### CODE REVIEW

**Comment by stevenbrix** [comment]
@rusamentiaga can you resolve the conflicts? then we can get this merged. sorry for the delay!

**Comment by rusamentiaga** [comment]
> 
> 
> @rusamentiaga can you resolve the conflicts? then we can get this merged. sorry for the delay!

done

File: src/Microsoft.DotNet.Wpf/src/WindowsBase/System/Security/Permissions/MediaPermission.cs [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    }
+    public enum MediaPermissionImage
+    {
+        NoImage,
+        SiteOfOriginImage,
+        SafeImage,
+        AllImage,
+    }
+    [Serializable()]
+    sealed public class MediaPermission : CodeAccessPermission, IUnrestrictedPermission
```
Comment (weltkante) [inline src/Microsoft.DotNet.Wpf/src/WindowsBase/System/Security/Permissions/MediaPermission.cs]: re-adding this looks like a mistake while merging?

File: src/Microsoft.DotNet.Wpf/src/WindowsBase/System/Security/Permissions/WebBrowserPermission.cs [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+{
+    public enum WebBrowserPermissionLevel
+    {
+        None,
+        Safe,
+        Unrestricted
+     }
+    
+    [Serializable()]
+    sealed public class WebBrowserPermission : CodeAccessPermission, IUnrestrictedPermission
```
Comment (weltkante) [inline src/Microsoft.DotNet.Wpf/src/WindowsBase/System/Security/Permissions/WebBrowserPermission.cs]: re-adding this looks like a mistake while merging?

Comment (rusamentiaga) [reply src/Microsoft.DotNet.Wpf/src/WindowsBase/System/Security/Permissions/WebBrowserPermission.cs]: Sorry, my mistake

**Comment by ryalanms** [comment]
Thanks, @rusamentiaga.

**Pull request merged**